guidedmodules/models.py

```diff
@@ -120,7 +120,7 @@ class AppVersion(models.Model):
     version_number = models.CharField(blank=True, null=True, max_length=128, help_text="The version number of the compliance app.")
     version_name = models.CharField(blank=True, null=True, max_length=128, help_text="The name of this version/release of the compliance app.")
 
-    input_files = models.ManyToManyField('guidedmodules.AppInput', blank=True, null=True, help_text="The inputs linked to this pack.")
+    input_files = models.ManyToManyField('guidedmodules.AppInput', blank=True, help_text="The inputs linked to this pack.")
```

Comment:
Are we sure we want to remove `null=True`?

Reply:
Yes. It is not used. That is the point of the migration